### PR TITLE
Add a code of conduct to phlowers

### LINKED ISSUE #5: Add a code of conduct
### Describe the current behavior

_No response_

### Describe the expected behavior

A code of conduct should be added

### Describe the motivation

_No response_

### Extra Information

cf. #2

### PR DESCRIPTION
**Pull request by adriengoeller**
**Please check if the PR fulfills these requirements**

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

Fixes #5 

**What kind of change does this PR introduce?**

Project document setup : this PR adds a code of conduct

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [x] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *breaking change* or *deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Phlowers Code of Conduct
+
+
+## Introduction
+
+This Code of Conduct applies to all spaces managed by the NumPy project, including all public and private mailing lists, issue trackers, wikis, blogs, Twitter, and any other communication channel used by our community. The NumPy project does not organise in-person events, however events related to our community should have a code of conduct similar in spirit to this one.
```
Comment (annetill) [inline CODE_OF_CONDUCT.md:6]: You cannot make a copy past, it should remain an inspiration. First, you have many reference to the Numpy project. And then, I think you should remain "simple" : we just have a slack and a github (issues, PRs). We are not particpating in events right now.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - Posting sexually explicit or violent material.
+    - Posting (or threatening to post) other people’s personally identifying information (“doxing”).
+    - Sharing private content, such as emails sent privately or non-publicly, or unlogged forums such as IRC channel history, without the sender’s consent.
+    - Personal insults, especially those using racist or sexist terms.
+    - Unwelcome sexual attention.
+    - Excessive profanity. Please avoid swearwords; people differ greatly in their sensitivity to swearing.
+    - Repeated harassment of others. In general, if someone asks you to stop, then stop.
+    - Advocating for, or encouraging, any of the above behaviour.
+
+## Diversity Statement
```
Comment (annetill) [inline CODE_OF_CONDUCT.md:32]: I think diversity statement is not necessary at this stage of the projet. But we can say somewhere that we are communicating in English.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+Standards for behaviour in the NumPy community are detailed in the Code of Conduct above. Participants in our community should uphold these standards in all their interactions and help others to do so as well (see next section).
+
+## Reporting Guidelines
+
+As the project is not ready to accept external contribution for the moment, this section will be completed later.
+
+## Endnotes
+
+We are thankful to the groups behind the following documents, from which we drew content and inspiration:
```
Comment (annetill) [inline CODE_OF_CONDUCT.md:47]: Great to mention.

**Comment by annetill** [comment]
I suggest something really simple because we are not open to external collaborations right now and copy-past does not respect copyright, such as: 

**Code of Conduct for Phlowers**

Welcome to the Phlowers project! We aim to foster a welcoming, collaborative, and respectful community. By participating in this project, you agree to adhere to the following principles.

**Specific Guidelines**

Be Open and Inclusive.
Everyone is welcome to contribute to Phlowers. Use public channels for discussions unless confidentiality is necessary. Public conversations encourage transparency and collective problem-solving.

Be Respectful and Kind.
Treat others with empathy, patience, and kindness. Disagreements are natural, but personal attacks, threats, or derogatory comments are not tolerated. Assume good intentions and strive for constructive resolutions.

Foster Collaboration.
Our collective success depends on sharing knowledge and building on each other’s contributions. Be ready to explain your work to help others learn and improve the project.

Encourage Learning and Curiosity.
No one has all the answers. Questions are essential to growth—ask them freely and answer them thoughtfully. Redirect questions to the appropriate resources when needed.

Communicate Thoughtfully.
Choose your words carefully. Avoid harmful language and respect differences. Phlowers will not tolerate:

- Threats or violent language.
- Discriminatory remarks, including those based on race, gender, or any other identity.
- Sharing explicit, violent, or sensitive personal content without consent.
- Personal insults or targeted harassment.
- Encouraging or engaging in harmful behavior.

Be Mindful of Boundaries.
If someone asks you to stop a behavior, stop immediately. Respect personal and professional boundaries to maintain a safe and supportive environment.

By following this Code of Conduct, we ensure that Phlowers remains a space where creativity, collaboration, and growth can thrive. Thank you for contributing to our community!

**Comment by adriengoeller** [comment]
Ok @annetill thank you for your proposition.
I suggest that I integrate it in place of the review.
I let you resolve the points from your first review ?

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+No one has all the answers. Questions are essential to growth—ask them freely and answer them thoughtfully. Redirect questions to the appropriate resources when needed.
+
+**Communicate Thoughtfully.**
+
+Choose your words carefully. Avoid harmful language and respect differences. Phlowers will not tolerate:
+
+**Threats or violent language.**
+
+Discriminatory remarks, including those based on race, gender, or any other identity.
```
Comment (annetill) [inline CODE_OF_CONDUCT.md:29]: Typo here, end of line character missing (two spaces).

**Review by annetill — APPROVED** [review]

**Review by annetill — APPROVED** [review]

**Review by annetill — APPROVED** [review]

**Review by annetill — APPROVED** [review]

**Pull request merged**